sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -837,6 +861,7 @@ public partial class IndexerExecutionResult
         public int ItemCount { get { throw null; } }
         public System.DateTimeOffset? StartTime { get { throw null; } }
         public Azure.Search.Documents.Indexes.Models.IndexerExecutionStatus Status { get { throw null; } }
+        public Azure.Search.Documents.Indexes.Models.IndexerExecutionStatusDetail? StatusDetail { get { throw null; } }
```

Comment:
What does this tell us?  How to interpret `Running`?  Can we think of a better name?

Reply:
I am thinking we work with the service team to arrive at better names and update in the next preview.